src/build/arg/mod.rs

```diff
@@ -2608,11 +2631,11 @@ impl<'help> Arg<'help> {
         mut self,
         arg_id: T,
         val: Option<&'help OsStr>,
-        default: &'help OsStr,
+        default: impl Into<Option<&'help OsStr>>,
     ) -> Self {
         let l = self.default_vals_ifs.len();
         self.default_vals_ifs
-            .insert(l, (arg_id.into(), val, default));
+            .insert(l, (arg_id.into(), val, default.into()));
```

Comment:
We shouldn't need to do `.into()`. Let's restrict the conversion to a single place.

Reply:
I'm not sure what you mean here. The `Arg::default_vals_ifs` stores `Option<&'help OsStr>`, so don't you need the `into()`?